### PR TITLE
doc: fix v1 FragmentStatuses docs

### LINKED ISSUE #3063: Wrong API description for /v1/fragments/statuses
The [FragmentStatuses](https://github.com/input-output-hk/jormungandr/blob/c18270fecb5d0a90a799ea14218cd8440156fc2c/doc/api/v1.yaml#L49-L49) method description seems wrong: the method should be `GET`, the parameter is in the query string, and it's a comma-separated list and not an array.

### PR DESCRIPTION
**Pull request by eugene-babichenko**
fixes #3063

### CODE REVIEW

File: doc/api/v1.yaml [hunk 1]
```diff
@@ -46,23 +46,19 @@ paths:
...
-                pattern: '[0-9a-f]+'
-            example: ["68dcc12fe0dfe5e7b66ca6f8c959f9aa43b273e120a77fc3e4e2f04f1ecd7968"]
-          
+      parameters:
+        - in: path
+          name: fragment_ids
+          description: Comma-separated list of fragment IDs
+          schema:
+            type: string
+            pattern: '[0-9a-f]+(,[0-9a-f]+)+'
```
Comment (mzabaluev) [inline doc/api/v1.yaml:60]: I wonder how common this format is in our API. What is an idiomatic way to pass an array of simple values in REST/OpenAPI?

File: doc/api/v1.yaml [hunk 2]
```diff
@@ -46,23 +46,19 @@ paths:
...
-            schema:
-              type: array
-              items:
-                type: string
-                pattern: '[0-9a-f]+'
-            example: ["68dcc12fe0dfe5e7b66ca6f8c959f9aa43b273e120a77fc3e4e2f04f1ecd7968"]
-          
+      parameters:
+        - in: path
+          name: fragment_ids
```
Comment (mzabaluev) [inline doc/api/v1.yaml:56]: Seems like the linter wants this to be in the endpoint path.

File: doc/api/v1.yaml [hunk 3]
```diff
@@ -46,23 +46,19 @@ paths:
...
-        description: List of fragment IDs
-        required: true
-        content:
-          application/json:
-            schema:
-              type: array
-              items:
-                type: string
-                pattern: '[0-9a-f]+'
-            example: ["68dcc12fe0dfe5e7b66ca6f8c959f9aa43b273e120a77fc3e4e2f04f1ecd7968"]
```
Comment (mzabaluev) [inline doc/api/v1.yaml:64]: Would be good to provide an updated example.

File: doc/api/v1.yaml (hunk 1 above)
Comment (eugene-babichenko) [reply doc/api/v1.yaml:60]: I am not sure we have it in other places. The problem is that it was the only way to pass an array of these as an argument. Initially I wanted to use repeated `fragment_id`, but I iirc it just did was not possible.

File: doc/api/v1.yaml (hunk 2 above)
Comment (eugene-babichenko) [reply doc/api/v1.yaml:56]: Ah, that is incorrect. It is not in path, but in query.

File: doc/api/v1.yaml (hunk 3 above)
Comment (eugene-babichenko) [reply doc/api/v1.yaml:64]: Done.

File: doc/api/v1.yaml (hunk 1 above)
Comment (mzabaluev) [reply doc/api/v1.yaml:60]: Query string does not forbid repeating keys AFAIK (worked with PHP array args in 2001!), but this way is more compact.

**Review by mzabaluev — APPROVED** [review]

File: doc/api/v1.yaml (hunk 1 above)
Comment (eugene-babichenko) [reply doc/api/v1.yaml:60]: This was the problem with `warp`, not the standard restricting me from doing that.

**Pull request merged**